File: programmers.co.kr/2016년/solution.cpp
#include <string>
#include <vector>

using namespace std;

int calender[12] = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
string Days[7] = {"FRI","SAT","SUN", "MON","TUE","WED","THU"};

string solution(int a, int b)
{
    string answer = "";
    int count = 0;
    for(int i = 0; i < a - 1; i++){
        count += calender[i];
    }
    count += (b - 1);
    return Days[count % 7];
}

File: programmers.co.kr/2016년/solution.py
def solution(a, b):
    answer = ''
    calender = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    week_list = ["FRI","SAT","SUN", "MON","TUE","WED","THU"]
    count = 0
    for i in range(a - 1):
        count += calender[i]
    count += (b - 1)
    answer = week_list[count % 7]
    return answer